pages/linux/lastcomm.md

```diff
@@ -0,0 +1,20 @@
+# lastcomm
+
+> Show last commands executed.
+> More information: <https://manpages.debian.org/stable/acct/lastcomm.1.en.html>.
+
+- Print informations about all of the commands in the acct (record file):
+
+`lastcomm`
+
+- Display commands executed by user:
+
+`lastcomm --user {{user}}`
+
+- Display informations about command executed on the system:
+
+`lastcomm --command {{command}}`
+
+- Display informations about terminal_name activities executed on the system:
```

Comment:
Display information about commands executed on a given terminal

Reply:
done in commit 8d8e56f3a2054eeb9191c724e23142ea36d300a1